src/Compilers/VisualBasic/Portable/Lowering/Rewriter.vb

```diff
@@ -14,6 +15,9 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
             body As BoundBlock,
             previousSubmissionFields As SynthesizedSubmissionFields,
             compilationState As TypeCompilationState,
+            instrumentForDynamicAnalysis As Boolean,
+            ByRef dynamicAnalysisSpans As ImmutableArray(Of SourceSpan),
```

Comment:
Is this parameter meant to be used as an "out" parameter? Consider explicitly marking it with an \<Out\> attribute and make sure it is assigned before function returns.